build.sbt

```diff
@@ -93,6 +93,53 @@ ThisBuild / headerLicense  := Some(HeaderLicense.Custom(
      |""".stripMargin
 ))
 
+ThisBuild / develocityConfiguration := {
+  def pubDev = Option(System.getProperty("scala.build.publishDevelocity"))
```

Comment:
Not really clear what values this property can contain.

Reply:
Right, I'll add a comment.